### PR TITLE
NT Painkiller Litany Buff

### PR DESCRIPTION
**Pull request by assortedbeads**
## About The Pull Request

The NT painkiller litany now a version of paracetamol to you that is
- un-scannable 
- has no nsa
- is hard to overdose on

## Why It's Good For The Game

The NT litany was too weak to be of any use before. Compared to FBP's who can fix away their pain with it, there weren't many advantages to being a NT outside the church.

While this is just a small change, hopefully it will not only make up for being locked out of augument and mutations, it will make the pain killer litany a tactical option to use.

## Testing

- Launched the test server
- Spawned in as a NT vagabond
- Teleported to the round end area
- Attacked myself with a crowbar till my leg broke
- Did the litany
- Opened up the variables tab
- Checked to see the reagent was in my bloodstream
- Checked to see the reagent was being processed properly

## Changelog
:cl:
balance: NT pain litany now adds angelsbalm to your bloodstream
balance: NT pain litany is shorter to say now
/:cl:

### CODE REVIEW

File: code/modules/core_implant/cruciform/rituals/base.dm [hunk 1]
```diff
@@ -17,13 +17,13 @@
...
 	name = "Relief"
-	phrase = "Et si ambulavero in medio umbrae mortis non timebo mala"
+	phrase = "Semper invicta"
 	desc = "Short litany to relieve pain of the afflicted."
 	power = 20
 	ignore_stuttering = TRUE
 
 /datum/ritual/cruciform/base/relief/perform(mob/living/carbon/human/H, obj/item/implant/core_implant/C)
-	H.add_chemical_effect(CE_PAINKILLER, 10)
+	H.add_chemical_effect(CE_PAINKILLER, 50)
```
Comment (Humonitarian) [inline code/modules/core_implant/cruciform/rituals/base.dm:26]: This doesn't actually work as intended.

Chem effects are all removed on life tick, so this actually only lasts for about 2 seconds. We should do something about this, probably.

Comment (assortedbeads) [reply code/modules/core_implant/cruciform/rituals/base.dm:26]: Is there anything I can do in this pr?

Comment (assortedbeads) [reply code/modules/core_implant/cruciform/rituals/base.dm]: Changed it to add to the blood stream as suggested

**Review by SirRichardFrancis — COMMENTED** [review]
Holy tramadol, lmao.
No NSA too. It could be removed via sleeper, btw.
And why add blood?

**Comment by assortedbeads** [comment]
> Holy tramadol, lmao.
> No NSA too. It could be removed via sleeper, btw.
> And why add blood?

Ah I honestly missed that part in the code, I can nix the blood part.

Is there a way to make it unremovable by sleeper? I dont mind if it is but just wondering

**Comment by DiaFRAME444** [comment]
Can holyinaprov also be removed by sleeper? If not, why so?

**Comment by SirRichardFrancis** [comment]
> Is there a way to make it unremovable by sleeper? I dont mind if it is but just wondering

```
/datum/reagent/medicine/tramadol/holy/trans_to_obj()
	return
```

> Can holyinaprov also be removed by sleeper? If not, why so?

Yes. Nobody bothered to, pardon, "milk" disciples for chemical as weak as inaprovaline. Tramadol is another story...

**Review by Pink-Chink — APPROVED** [review]

**Pull request merged**